lib/rstuf.rb

```diff
@@ -0,0 +1,9 @@
+module Rstuf
+  mattr_accessor :base_url
+  mattr_accessor :enabled, default: false
+  mattr_accessor :wait_for, default: 1
```

Comment:
This is in here for now for testing. RSTUF works async, once API call to add artifact is done, remote job is scheduled at RSTUF so the result is not available immediately. This is subject to test and to find some reasonable delay. I have kicked-off with 1 second. Most likely this will be later controlled by ENV var and configured looking at average delay time per environment.